modules/helfi_ckeditor/config/install/filter.format.full_html.yml

```diff
@@ -21,7 +21,7 @@ filters:
     status: true
     weight: -50
     settings:
-      allowed_html: '<br> <p class> <h2> <h3> <h4> <h5> <h6> <cite> <dl> <dt> <dd> <blockquote role aria-* cite class> <footer class> <div role aria-* class> <span role aria-* class dir lang> <ul type class> <ol type start> <strong> <em> <s> <sub> <sup> <a href aria-label title class id target="_blank" rel data-entity-type data-entity-uuid data-entity-substitution> <li> <table> <tr> <td rowspan colspan> <th rowspan colspan> <thead> <tbody> <tfoot> <caption>'
+      allowed_html: '<br> <p class> <h2 id> <h3 id> <h4 id> <h5 id> <h6 id> <dl> <dt> <dd> <blockquote role aria-* cite class> <div role aria-* class> <span role aria-* class dir> <ul type> <ol type start> <strong> <em> <s> <sub> <sup> <a href data-entity-type data-entity-uuid data-entity-substitution aria-label title class id target="_blank" rel> <li> <table> <tr> <td rowspan colspan> <th rowspan colspan> <thead> <tbody> <tfoot> <caption> <footer class> <cite>'
```

Comment:
Do we need to allow id attribute for the headings? I think this was intentionally removed so that the TOC-javascript can handle the heading ids 🤔